comps/llms/text-generation/vllm/ray/dependency/launch_vllmray.sh

```diff
@@ -30,14 +30,14 @@ fi
 docker run -d --rm \
     --name="vllm-ray-service" \
     --runtime=habana \
-    -v $PWD/data:/data \
+    -v $DATA_DIR:/data \
```

Comment:
This `DATA_DIR` env var will be provided by the user before running this script (which is mentioned in the README)